<?xml version="1.0"?>
<launch>
    <!-- <include file="$(find rtabmap_ros)/launch/rtabmap.launch"> <arg name="rtabmap_args" value=""/>
		<arg name="approx_sync" value="true"/>
        <arg name="queue_size" value="20"/>

        <arg name="stereo" value="true"/>
        <arg name="left_image_topic" value="/stereo/left/image_rect"/>
        <arg name="right_image_topic" value="/stereo/right/image_rect"/>
        <arg name="right_camera_info_topic" value="/stereo/right/camera_info"/>
        <arg name="left_camera_info_topic" value="/stereo/left/camera_info"/>
        <arg name="odom_topic" value="/mavros/odometry/out"/>

        <arg name="subscribe_scan_cloud" value="true"/>
        <arg name="scan_cloud_topic" value="/stereo/points2"/>

    </include> -->
    <!-- Argument for device id -->
    <!-- <arg name="leftcam" value="0"/>
    <arg name="rightcam" value="1"/> -->
	<!-- Our node to control the drone -->
	<!-- <node pkg="offboard_py" type="challenge_task_3.py" name="rob498_drone_05" required="true" output="screen" /> -->
    <node pkg="offboard_py" type="stereo_img_proc.py" name="camera">
        <!-- <remap from="undistorted1/left/image_raw" to="stereo/left/image_raw"/>
        <remap from="undistorted2/right/image_raw" to="stereo/right/image_raw"/>
        <remap from="undistorted1/camera_info" to="stereo/left/camera_info"/>
        <remap from="undistorted2/camera_info" to="stereo/right/camera_info"/> -->
    </node>
    <!--Subscriber node-->
    <node ns="stereo" pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" output="screen">
	    <param name="disparity_range" value="32" />
	    <param name="speckle_size" value="1000" />
	    <param name="approximate_sync" value="true"/>
    </node>    


   
</launch>
